coil-base/src/androidTest/java/coil/EventListenerTest.kt

```diff
@@ -90,9 +87,9 @@ class EventListenerTest {
             imageLoader.testEnqueue {
                 data("$SCHEME_ANDROID_RESOURCE://${context.packageName}/${R.drawable.normal}")
                 transformations(object : Transformation {
-                    override fun key() = "test_transformation"
+                    override val cacheKey = "test_transformation"
```

Comment:
May should declare their types in override methods and properties ?

Reply:
IMO it's probably ok to avoid specifying the type here since it's for a local anonymous class.